packages/zowe-explorer/src/command/TsoCommandHandler.ts

```diff
@@ -298,7 +297,14 @@ export class TsoCommandHandler extends ZoweCommandProvider {
             }
         }
         if (tsoProfile) {
-            iStartTso.forEach((p) => (tsoParms[p] = tsoProfile.profile[p]));
+            tsoParms = {
+                ...iStartTso.reduce((obj, parm) => {
+                    const newObj = obj;
+                    newObj[parm] = tsoProfile.profile[parm];
+
+                    return newObj;
```

Comment:
Could these 3 lines be simplified to `return { ...obj, [parm]: tsoProfile.profile[parm] };`?